packages/reports/src/Components/indicative-cell.tsx

```diff
@@ -7,20 +8,28 @@ type TIndicativeCell = {
     amount: number;
     contract_info: TContractInfo;
     currency: string;
-    status?: string;
     is_footer: boolean;
     is_sell_requested: boolean;
+    profit: string;
+    status?: string;
 };
 
 const IndicativeCell = observer((props: TIndicativeCell) => {
-    const { amount, contract_info, currency, is_footer, is_sell_requested } = props;
+    const { amount, contract_info, currency, is_footer, is_sell_requested, profit } = props;
     const { portfolio } = useStore();
     const { onClickSell } = portfolio;
 
     return (
         <div className='open-positions__indicative'>
             <div className='open-positions__indicative--amount'>
-                <Money amount={Math.abs(amount)} currency={currency} />
+                <div
+                    className={classNames({
+                        'dc-contract-card--profit': Number(profit) > 0,
+                        'dc-contract-card--loss': Number(profit) < 0,
+                    })}
+                >
+                    <Money amount={Math.abs(amount)} currency={currency} />
+                </div>
                 <ArrowIndicator value={amount} />
```

Comment:
Sorry for silly question, but in other places you added a condition for `ArrowIndicator` rendering (`!is_sold`). Is there a need to add it here too? Please, ignore if it's not relevant as I don't know details

Reply:
No problem, it's not needed here because when the contract is sold, the whole row including this component disappears from the page. Therefore, there's no need to add this condition.

Reply:
And it was a good question 👍🏻